core/server/apps/private-blogging/lib/views/private.hbs

```diff
@@ -29,24 +29,29 @@
                                     <h1>This blog is private</h1>
                                 </header>
                                 <form class="gh-signin" method="post" novalidate="novalidate">
-                                    <div class="form-group">
-                                        <span class="input-icon icon-lock">
-                                            {{input_password class="gh-input" placeholder="Password"}}
-                                        </span>
-                                    </div>
+                                    {{#if error}}
+                                        <div class="form-group error">
```

Comment:
I think this could be simplified by by using the conditional to output the `.error` class rather than duplicating the block?

```handlebars
<div class="form-group{{#if error}} error{{/if}}">
```